File: openapi/code_samples/python/api_company_{cui}_financial_indicators.py
from flask import Flask, request, jsonify

app = Flask(__name__)

@app.route('/api/company/list-indicators/<string:cui>', methods=['GET'])
def list_indicators(cui):
    currency = request.args.get('currency', 'RON')
    # Your implementation here
    return jsonify({}), 200

if __name__ == '__main__':
    app.run()

File: openapi/code_samples/csharp/api_company_{cui}_financial_indicators.cs
using Microsoft.AspNetCore.Mvc;
using Swashbuckle.AspNetCore.Annotations;

[ApiController]
[Route("api/[controller]")]
public class CompanyController : ControllerBase
{
    [HttpGet]
    [Route("list-indicators/{cui}")]
    [SwaggerOperation(Summary = "Retrieves the list of financial indicators for a given company CUI.")]
    [ProducesResponseType(typeof(FinancialIndicatorResponseDTO), 200)]
    [ProducesResponseType(typeof(Fault), 400)]
    public IActionResult ListIndicators([FromRoute] string cui, [FromQuery(Name = "currency")] string currency = "RON")
    {
        // Your implementation here
        return Ok();
    }
}
